src/btree/bt_read.c

```diff
@@ -431,11 +401,8 @@ __instantiate_lookaside(WT_SESSION_IMPL *session, WT_REF *ref)
             las_prepare_cnt++;
         }
 
-        if (birthmark_record) {
-            birthmarkp->instantiated = true;
-            birthmark_record = false;
-        }
-
+        page_las->birthmarks[i].instantiated = true;
```

Comment:
check if we need `instantiated` now. Also we do not need `__instantiate_birthmarks()` further down at line 414

Reply:
535a348